soc/arm/nordic_nrf/Kconfig.defconfig

```diff
@@ -7,22 +7,21 @@ if SOC_FAMILY_NRF
 
 source "soc/arm/nordic_nrf/*/Kconfig.defconfig.series"
 
-# If the kernel has timer support, enable both clock control and timer
+# If the kernel has timer support, enable clock control
 if SYS_CLOCK_EXISTS
 
 config CLOCK_CONTROL
 	default y
 
-config NRF_RTC_TIMER
-	default y
-
 endif # SYS_CLOCK_EXISTS
 
 config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 1000000 if NRF_GRTC_TIMER
 	default 32768
 
 config SYS_CLOCK_TICKS_PER_SEC
 	default 128 if !TICKLESS_KERNEL
+	default 10000  if NRF_GRTC_TIMER
```

Comment:
maybe `!TICKLESS_KERNEL` variant for GRTC should also get a different ticks per sec? 100?